server/config.yml

```
@@ -0,0 +1,3 @@
+auth:
+  enabled: false
+  providers:
```

good. lets leave some comments in here, example

```suggestion
  providers:
  	  ## example provider config
	  # - label: 'googleoidc'
	  #   type: oidc
	  #   issuer: https://accounts.google.com
	  #   client_id: xxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxx.apps.googleusercontent.com
	  #   client_secret: xxxxxxxxxxxxxxxxxxxxxxxx
	  #   callback_base_uri: http://localhost:8088
```